test/cargoQueue.js

```diff
@@ -46,7 +46,7 @@ describe('cargoQueue', () => {
                 expect(c.length()).to.equal(0);
                 call_order.push('callback ' + 3);
             });
-        }, 15);
+        }, 20);
```

Comment:
So even if we don't introduce a test variable that acts as a multiplier for all/most timeout values, I would still increase these hard-coded values in a consistent way so it's easier to explain in the PR and doesn't feel like us arbitrarily picking values that work in our CI environments. For example, you multiplied each value in `delays` by 2 (which sounds reasonable), but then only increased this one by 1.3 (15 * 1.3), and the next by 2.67 (30 * 2.67).

I think it will be easier for them to accept these changes if the changes feel more uniform versus us finding values that happen to work in our CI system. Does that make sense?

Reply:
I see your point. Change here was intended to make timing diff more significant, so one step happens earlier and the next happens later, mitigating race conditions.
Anyway, I'm rethinking all my timing changes.

Reply:
When I try more systematic changes on timings, I get worse results. See this run:
https://dev.azure.com/pablo-nunez/Runner/_build/results?buildId=215&view=ms.vss-test-web.build-test-results-tab
Doubling delays doesn't fix `queue basics` (2 failures in OSX). This is because nothing enforces the order of execution of processes and callbacks, so we're subject to race conditions if delays are not significant enough. Their original delays are a bit random, and on my proposal I'm exaggerating them to enforce relative order. See better results on this branch:
https://dev.azure.com/pablo-nunez/Runner/_build/results?buildId=214&view=ms.vss-test-web.build-test-results-tab
So what's pending:
- Identify other tests that should be tweaked up.
- Investigate timeouts in Windows 10 browser tests.
- Fix autoInject for Safari in production code.